src/aks-preview/azext_aks_preview/tests/latest/recordings/test_aks_create_using_azurecni_with_pod_identity_enabled.yaml

```diff
@@ -403,60 +637,24 @@ interactions:
       - --resource-group --name --location --generate-ssh-keys --enable-managed-identity
         --enable-pod-identity --network-plugin
       User-Agent:
-      - python/3.8.6 (macOS-10.16-x86_64-i386-64bit) msrest/0.6.18 msrest_azure/0.6.3
-        azure-mgmt-containerservice/11.0.0 Azure-SDK-For-Python AZURECLI/2.14.2
+      - python/3.7.3 (Linux-4.15.0-1111-azure-x86_64-with-debian-10.2) msrest/0.6.21
+        cloud-shell/1.0 msrest_azure/0.6.3 azure-mgmt-containerservice/11.0.0 Azure-SDK-For-Python
+        AZURECLI/2.21.0
     method: GET
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest000001/providers/Microsoft.ContainerService/managedClusters/cliakstest000002?api-version=2021-02-01
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.ContainerService/locations/westus2/operations/1aa33b59-2782-4a10-8c11-19e185f060c8?api-version=2016-03-30
```

Comment:
why we need this change? the old uri before is to get an AKS cluster using 2021-02-01, the new url is to list operations using 2016-03-30.

Reply:
I think that s some misalignment issue, all I did is to change the version from 2021-02-01 to 2021-03-01 and then rerun the test.